openpower/package/openpower-pnor/Config.in

```diff
@@ -35,6 +35,17 @@ config BR2_SKIBOOT_LID_NAME
         help
             String used to define sapphire lid filename
 
+config BR2_TARGET_SKIBOOT_XZ
+        boolean "Compress the skiboot image with XZ"
+        default n
+
+config BR2_SKIBOOT_LID_XZ_NAME
+        string "Name of compressed skiboot lid name to be used"
+        default "skiboot.lid" if !BR2_TARGET_SKIBOOT_XZ
+        default "skiboot.lid.xz" if BR2_TARGET_SKIBOOT_XZ
+        help
+            String used to define compressed sapphire lid filename
```

Comment:
should be skiboot rather than Sapphire.

Sapphire was only ever an internal name.